File: hackathon-backend/src/main/java/hackathon/entity/Employee.java
package hackathon.entity;

import javax.persistence.*;
import javax.validation.constraints.NotNull;

@Entity
@Table(name = "employee")
public class Employee {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id = 0L;

    @NotNull
    @Column(name = "name", nullable = false)
    private String name;

    @NotNull
    @Column(name = "about_me", nullable = false)
    private String aboutMe;

    @NotNull
    @Column(name = "tags", nullable = true)
    private String tags;

//    public Employee(){}

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAboutMe() {
        return aboutMe;
    }

    public void setAboutMe(String aboutMe) {
        this.aboutMe = aboutMe;
    }

    public String getTags() {
        return tags;
    }

    public void setTags(String tags) {
        this.tags = tags;
    }



}

File: hackathon-backend-kotlin/src/main/kotlin/hackathon/backend/kotlin/entity/Employee.kt
package hackathon.backend.kotlin.entity

import hackathon.Repository.EmployeeRepository
import javax.persistence.*
import javax.validation.constraints.NotNull

@Entity
@Table(name = "employee")
@NamedQueries( NamedQuery(name = EmployeeRepository.Queries.FindAllEmployees.NAME,
query = EmployeeRepository.Queries.FindAllEmployees.JPQL),
NamedQuery(name = EmployeeRepository.Queries.FindByName.NAME,
query = EmployeeRepository.Queries.FindByName.JPQL))
data class Employee (
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long = 0L,

    @NotNull
    @Column(name = "name", nullable = false)
    var name: String,

    @NotNull
    @Column(name = "about_me", nullable = false)
    var aboutMe: String,

    @NotNull
    @Column(name = "tags", nullable = true)
    var tags: String
)
